src/schemas/21P-0969-overflow/example.json

```diff
@@ -23,47 +23,231 @@
+      },
+      {
+        "originalOwnerRelationship": "CHILD",
+        "otherOriginalOwnerRelationshipType": "Step-child",
+        "transferMethod": "TRADED",
+        "otherTransferMethod": "",
+        "assetType": "Artwork",
+        "newOwnerName": "Michael Green",
+        "newOwnerRelationship": "Cousin",
+        "saleReportedToIrs": true,
+        "transferDate": "2021-09-21",
+        "assetTransferredUnderFairMarketValue": false,
+        "fairMarketValue": 5000.0,
+        "saleValue": 5000.0,
+        "capitalGainValue": 2000.0
+      }
+    ],
+    "trusts": [
+      {
+        "establishedDate": "2020-03-15",
+        "marketValueAtEstablishment": 100000.0,
+        "trustType": "REVOCABLE",
+        "addedFundsAfterEstablishment": true,
+        "addedFundsDate": "2021-05-10",
+        "addedFundsAmount": 5000.0,
+        "receivingIncomeFromTrust": true,
+        "annualReceivedIncome": 12000.0,
+        "trustUsedForMedicalExpenses": false,
+        "trustEstablishedForVeteransChild": false,
+        "haveAuthorityOrControlOfTrust": true
+      },
+      {
+        "establishedDate": "2018-09-25",
+        "marketValueAtEstablishment": 50000.0,
+        "trustType": "IRREVOCABLE",
+        "addedFundsAfterEstablishment": false,
+        "receivingIncomeFromTrust": false,
+        "trustUsedForMedicalExpenses": true,
+        "monthlyMedicalReimbursementAmount": 300.0,
+        "trustEstablishedForVeteransChild": true,
+        "haveAuthorityOrControlOfTrust": false
+      },
+      {
+        "establishedDate": "2022-01-01",
+        "marketValueAtEstablishment": 75000.0,
+        "trustType": "BURIAL",
+        "addedFundsAfterEstablishment": true,
+        "addedFundsDate": "2023-06-20",
+        "addedFundsAmount": 2000.0,
+        "receivingIncomeFromTrust": false,
+        "trustUsedForMedicalExpenses": false,
+        "trustEstablishedForVeteransChild": false,
+        "haveAuthorityOrControlOfTrust": true
+      }
+    ],
+    "annuities": [
+      {
+        "establishedDate": "2015-03-01",
+        "marketValueAtEstablishment": 150000.0,
+        "addedFundsAfterEstablishment": true,
+        "addedFundsAmount": 50000.0,
+        "addedFundsDate": "2018-06-15",
+        "revocable": false,
+        "receivingIncomeFromAnnuity": true,
+        "annualReceivedIncome": 6000.0,
+        "canBeLiquidated": false,
+        "surrenderValue": 100000.0
       },
       {
-        "recipientRelationship": "CUSTODIAN",
-        "recipientName": "Sam Doe",
-        "incomeType": "UNEMPLOYMENT",
-        "grossMonthlyIncome": 1000.00,
-        "payer": "Agency of Unemployment"
+        "establishedDate": "2010-05-22",
+        "marketValueAtEstablishment": 100000.0,
+        "addedFundsAfterEstablishment": false,
+        "revocable": true,
+        "receivingIncomeFromAnnuity": false,
+        "annualReceivedIncome": 0.0,
+        "canBeLiquidated": true,
+        "surrenderValue": 80000.0
       }
     ]
   }
-}
```

Comment:
Please add new line

Reply:
When I open this file in a code editor, there is a trailing empty line (254) after the new closing curly brace on line 253. Do you want to add another new line(255)?